docs/0.1/references/cli/grid-product.1.md

```diff
@@ -1,11 +1,75 @@
-% GRID-KEYGEN(1) Cargill, Incorporated | Grid Commands
+% GRID-PRODUCT(1) Cargill, Incorporated | Grid Commands
 <!--
   Copyright 2018-2020 Cargill Incorporated
   Licensed under Creative Commons Attribution 4.0 International License
   https://creativecommons.org/licenses/by/4.0/
 -->
 
-USAGE
+NAME
 ====
 
-**grid product** \[**FLAGS**\] <**SUBCOMMAND**>
+**grid-product** — Create, update, or delete Grid Products
+
+SYNOPSIS
+========
+
+**grid product** \[**FLAGS**\] \[**SUBCOMMAND**\]
+
+DESCRIPTION
+===========
+
+This command allows for the creation and management of Grid Products.
+Commands to list and display Product data are also available.
+
+FLAGS
+=====
+
+`-h`, `--help`
+: Prints help information
+
+`-k`, `--key`
+: Base name for private key file
+
+`-q`, `--quiet`
+: Do not display output
+
+`--service-id`
+: The ID of the service the payload should be sent to; required if running on
+  Splinter. Format <circuit-id>::<service-id>
+
+`-V`, `--version`
+: Prints version information
+
+`-v`
+: Increases verbosity (the opposite of `-q`). Specify multiple times for more
+  output
+
+`--url`
+: URL for the REST API
+
+`--wait`
+: How long to wait for transaction to be committed
+
+ENVIRONMENT VARIABLES
```

Comment:
A "mid-level" command like this should list the subcommands after the options.  See splinter-circuit.1.md for an example of a mid-level command:

https://github.com/Cargill/splinter-docs/blob/master/docs/0.4/references/cli/splinter-circuit.1.md
https://www.splinter.dev/docs/0.4/references/cli/splinter-circuit.1.html